### PR TITLE
feat: hybrid msgs

### LINKED ISSUE #2: Publish for ROS 2 foxy + galactic
I believe this package is currently available in ROS 1 (noetic). We should add it to ROS 2 (foxy + galactic) distros as well.

### PR DESCRIPTION
**Pull request by wep21**
**Public-Facing Changes**
Suppot ros2

**Description**
Closes #2

### CODE REVIEW

**Comment by wep21** [comment]
cc @jhurliman @amacneil

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,14 +1,64 @@
...
+  ament_export_dependencies(rosidl_default_runtime)
+
+  if(BUILD_TESTING)
+    find_package(ament_lint_auto REQUIRED)
+    ament_lint_auto_find_test_dependencies()
+  endif()
+
+  ament_package()
+
+endif()
```
Comment (wep21) [inline CMakeLists.txt]:
```suggestion
endif()

```

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,14 +1,64 @@
...
+set(ROS_VERSION $ENV{ROS_VERSION})
 
-generate_messages(DEPENDENCIES geometry_msgs std_msgs visualization_msgs)
+if(${ROS_VERSION} EQUAL 1)
 
-catkin_package(CATKIN_DEPENDS geometry_msgs message_runtime std_msgs visualization_msgs)
+  cmake_minimum_required(VERSION 2.8.3)
+
+  # Default to C++11
+    if(NOT CMAKE_CXX_STANDARD)
```
Comment (wep21) [inline CMakeLists.txt]:
```suggestion
  if(NOT CMAKE_CXX_STANDARD)
```

**Comment by wep21** [comment]
@jhurliman friendly ping

**Review by jhurliman — APPROVED** [review]

**Comment by wep21** [comment]
@jhurliman Could you merge and create a new tag?

**Pull request merged**